js/app/modules/arrangement/sideBySide.js

```diff
@@ -12,7 +12,7 @@ const Knowledge = imports.app.knowledge;
 const Module = imports.app.interfaces.module;
 const Utils = imports.app.utils;
 
-const SPACING_MAX = 50;
+const SPACING_MAX = 0;
```

Comment:
OK, I am asking because a good chunk of the work I did on the topbar was to have that responsive spacing between cards. I wish I knew beforehand :D
So, I think the ideal thing would be to replace that constant with a property with a default of 0, let me know if you want me to do it